### PR TITLE
adding in vertx cluster config

### PR DESCRIPTION
**Pull request by krmarcus-ucsd**
Wasn't exactly sure how to test this

### CODE REVIEW

File: module-info/pom.xml [hunk 1]
```diff
@@ -25,6 +25,21 @@
       <artifactId>vertx-web-client</artifactId>
       <version>${vertx.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-hazelcast</artifactId>
+      <version>3.7.0</version>
```
Comment (crawld) [inline module-info/pom.xml]: change to ${vertx.version}

File: resources/configurations/configuration.xml [hunk 2]
```diff
@@ -104,6 +104,10 @@
             <uri>http://sandbox.rohub.org/rodl/ROs/</uri>
         </roHub>
 
+        <!-- set true if deploying in kubernetes with dns discovery for hazelcast -->
+        <deployInKubernetes>false</deployInKubernetes>
+        <hazelcastDiscoveryK8sDnsService>hazelcast-service.wifire.svc.cluster.local</hazelcastDiscoveryK8sDnsService>
```
Comment (crawld) [inline resources/configurations/configuration.xml]: leave this one blank and change in firemap configuration.xml

File: module-info/pom.xml (hunk 1 above)
Comment (krmarcus-ucsd) [reply module-info/pom.xml]: done

File: resources/configurations/configuration.xml (hunk 2 above)
Comment (krmarcus-ucsd) [reply resources/configurations/configuration.xml]: took these out

**Pull request merged**